include/tvm/script/ir_builder/relax/frame.h

```diff
@@ -100,6 +100,11 @@ class FunctionFrameNode : public SeqExprFrameNode {
    * \sa ret_type
    */
   Optional<tvm::relax::Expr> ret_shape;
+  /*!
+   * \brief The function return struct info.
+   * \sa ret_type
+   */
+  Optional<tvm::relax::StructInfo> ret_sinfo;
```

Comment:
Perhaps we don’t need to “see also” here?
```suggestion
  /*!
   * \brief The function return struct info.
   */
  Optional<tvm::relax::StructInfo> ret_sinfo;
```

Reply:
This one looks better.
```suggestion
  /*! \brief The function return struct info. */
  Optional<tvm::relax::StructInfo> ret_sinfo;
```